src/Microsoft.AspNetCore.WebHooks.Receivers.Stripe/Filters/StripeTestEventRequestFilter.cs

```diff
@@ -71,7 +71,8 @@ public void OnResourceExecuting(ResourceExecutingContext context)
             if (IsTestEvent(notificationId))
             {
                 // Log about and optionally short-circuit this test event.
-                if (_configuration.IsTrue(StripeConstants.PassThroughTestEventsConfigurationKey))
+                var passThroughString = _configuration[StripeConstants.PassThroughTestEventsConfigurationKey];
+                if (bool.TryParse(passThroughString, out var passThrough) && passThrough)
```

Comment:
@rynowak I'm assuming you were objecting to the helper method. But, is this the syntax that makes your head hurt i.e. do you dislike everything but
``` c#
var passThroughString = _configuration[StripeConstants.PassThroughTestEventsConfigurationKey];
bool.TryParse(passThroughString, out var passThrough);
if (passThrough)
{
    // ...
}
```

Reply:
I am objecting the helper method.